osgar/replay.py

```diff
@@ -93,9 +94,12 @@ def main():
     module_instance = replay(args)
     module_instance.verbose = args.verbose
 
+    signal.signal(signal.SIGINT, lambda signum, frame: module_instance.request_stop())
     module_instance.start()
     # now wait until the module is alive
-    module_instance.join()
+    while module_instance.is_alive():
+        module_instance.join(0.2)
```

Comment:
this change unfortunately breaks normal replay, i.e. you have to use Ctrl+C always for complete as interrupted recorded logs